### ISSUE TITLE
Add a reply button for posts

### ISSUE THREAD

**Issue by coleGillespie** [bigboringsystem/bigboringsystem#105]
Instead of having the user copy and paste the reply url it would be nice to just click some reply text and would send you strait to the post page then automatically apply the reply post url in the post form

**Comment by isaacs** [comment]
Yes, please.

**Comment by ednapiranha** [comment]
whichever is easiest. so probably the two parts one works.

**Comment by coleGillespie** [comment]
@isaacs @ednapiranha Seems simple enough. Ill add something locally and test it.

**Comment by coleGillespie** [comment]
@isaacs @ednapiranha do we only want to see this on the post page? or do we also want to see this inside of the discussion feed?

**Comment by isaacs** [comment]
@thefoxis @brittanystoroz @selenamarie Any front-end folks wanna review this?

I accidentally assigned myself to it, and I guess github doesn't allow you to un-assign yourself, so... @thefoxis I choose YOUUUUUUU!!!!!!!11111

**Comment by coleGillespie** [comment]
@thefoxis did you just clone it for the first time? i get 404 when the for the first time see #103

**Comment by thefoxis** [comment]
@coleGillespie&mdash;woah, thanks! I looked at it and it looks good to me :ok_hand:

**Comment by coleGillespie** [comment]
@thefoxis thanks for the review

**Comment by coleGillespie** [comment]
@jasonrhodes just merged this. Thanks for the help everyone.

**Comment by ednapiranha** [comment]
This is actually breaking a lot of things. Next time there needs to be more substantial testing.
It breaks the following behavior:

1. Links do not link properly on replies
2. Multiple links are not adhered to
3. Comma/space-delimited links do not work

**Comment by coleGillespie** [comment]
@ednapiranha it looks like there was a fix put in for number one.

I am not sure i follow number two and three can you explain more?

**Comment by ednapiranha** [comment]
Turns out it wasn't twitter-text.

The problem was that your check was for port 80 and assumed that something that didnt follow with a port would always be port 80. We just switched to https which does not have a following port but is 443 not 80. So your logic would be invalid.

There was a subsequent check for a port which caused an error in reply display because it would do something like http://bbs.com:/blah where the colon would throw it off. I've fix it here https://github.com/bigboringsystem/bigboringsystem/commit/1f873e1505edc0ff76f538d8628864035783b07b#diff-dd78e4bcebf169bb21fd83745d573343R8
